### PR TITLE
SC-86 Asset Channels table incorrectly sized

### PR DESCRIPTION
**Pull request by KulinduC**
Used flex classes to properly size asset channel tables

### CODE REVIEW

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx [hunk 1]
```diff
@@ -140,16 +140,16 @@ const AssetChannelWindow = (props: IProps) => {
...
                 <div className="row">
                     <div className="col">
                         <h4>Channels:</h4>
                     </div>
                 </div>
             </div>
-            <div className="card-body">
-                <div style={{ width: '100%', maxHeight: window.innerHeight - 381, padding: 30 }}>
+            <div className="card-body" style={{ flex: 1, overflowY: 'auto' }}>
+                <div style={{ width: '100%', maxHeight: window.innerHeight - 381, padding: 10 }}>
```
Comment (gcsantos-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx:152]: We are trying to get rid of these window.innherHeight's, can you revise?

**Review by gcsantos-gpa — CHANGES_REQUESTED** [review]

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx (hunk 1 above)
Comment (KulinduC) [reply Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx:152]: Yes I can do that

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx [hunk 2]
```diff
@@ -140,16 +140,16 @@ const AssetChannelWindow = (props: IProps) => {
...
                 <div className="row">
                     <div className="col">
                         <h4>Channels:</h4>
                     </div>
                 </div>
             </div>
-            <div className="card-body">
-                <div style={{ width: '100%', maxHeight: window.innerHeight - 381, padding: 30 }}>
+            <div className="card-body" style={{ flex: 1, display: 'flex', flexDirection: 'column', overflowY: 'auto' }}>
+                <div style={{ flex: 1 }} >
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx:152]: This div is unnecesarry. the Table should take up `100%` of the parent (`card-body`) already

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx [hunk 3]
```diff
@@ -169,7 +169,7 @@ const AssetChannelWindow = (props: IProps) => {
                             }
                         }}
                         TheadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
-                        TbodyStyle={{ display: 'block', overflowY: 'scroll', maxHeight: window.innerHeight - 300, width: '100%' }}
+                        TbodyStyle={{ display: 'block', width: '100%' }}
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx:172]: see https://github.com/GridProtectionAlliance/SystemCenter/blob/ProcessedFileFix/Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ProcessedFile/ByFile.tsx#L212

**Review by clackner-gpa — CHANGES_REQUESTED** [review]

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx (hunk 3 above)
Comment (KulinduC) [reply Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx:172]: I'll be closing this PR and reference this ticket in SC-101

**Pull request closed without merging**